src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -466,15 +470,15 @@ public void refreshStudyInformation() {
             EdmApi edmApi = apiClient.getEdmApi();
             SearchApi searchApi = apiClient.getSearchApi();
 
-            String participantsEntitySetName = ChronicleServerUtil.getParticipantEntitySetName( studyId );
-            UUID participantsEntitySetId = edmApi.getEntitySetId( participantsEntitySetName );
-            if ( participantsEntitySetId == null ) {
+            String entitySetName = ChronicleServerUtil.getParticipantEntitySetName( studyId );
+            UUID entitySetId = edmApi.getEntitySetId( entitySetName );
+            if ( entitySetId == null ) {
                 logger.error( "Unable to load participant EntitySet id." );
                 return null;
             }
 
             List<NeighborEntityDetails> participantNeighbors = searchApi
-                    .executeEntityNeighborSearch( participantsEntitySetId, participantEntityId );
+                    .executeEntityNeighborSearch( entitySetId, participantEntityId );
```

Comment:
ok looks like this is the entityKeyId -- in that case, can you rename `participantEntityId` to `participantEntityKeyId` since entityId means something different?

Reply:
Done.